app/src/renderer/components/common/CookieBar.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div v-if="!session.cookiesAccepted" class="cookieBar">
+    <a class="close">
+      <i class="material-icons" @click="accept">close</i>
+    </a>
+    Lunie uses cookies and remote error collection to improve the app.
+    By dismissing this notification, you consent to this policy.
```

Comment:
does this comply to GDPR ?

Reply:
As far as I read yes. The user just needs to do an opt in action.